softdevice_controller/doc/softdevice_controller.rst

```diff
@@ -13,9 +13,6 @@ For the nRF53 Series, the requirements described in this document are only relev
 
 Applications utilizing the |controller| library are responsible for the following:
 
-* Forwarding RNG interrupts to the |controller| library.
```

Comment:
Should it be mentioned instead that the application is responsible for having an RNG driver and providing numbers to the controller when requested?